gcp-cloud-functions/sendgrid-email-scheduled-reminder/index.js

```diff
@@ -70,55 +78,85 @@ exports.sendgridEmailScheduledReminder = async (req, res) => {
 // [END cloud_tasks_func]
 
 // Function creates an HTML postcard with message.
-const reminderHTML = function (to_name) {
+const reminderHTML = function (vehicle_nickname, location, move_by_timestamp) {
   return `<html>
   <head>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
     <style>
+      body {
+        background: white;
+      }
+      
       .reminder {
-        width: 600px;
-        height: 400px;
-        background: #4285F4;
+        width: 80%;
+        margin: auto;
+        height: 100vh;
         text-align: center;
+        color: #484848;
       }
 
-      .reminder-text {
-        font-family: Arial, sans-serif;
-        font-size: 60px;
+      .reminder-header {
+        font-family: 'Open Sans', Arial, sans-serif;
+        font-size: 2.0em;
+        text-align: left;
         font-weight: bold;
-        text-transform: uppercase;
-        color: #FFF;
-        padding: 40px 0px;
+        overflow: hidden;
+        white-space: nowrap;
       }
 
-      .reminder-names {
-        font-family: Monaco, monospace;
-        font-size: 30px;
+      .reminder-text {
+        font-family: 'Open Sans', Arial, sans-serif;
+        font-size: 1.5em;
+        text-align: left;
+        padding: 30px 0px;
+      }
+
+      .reminder-text ul {
         text-align: left;
-        color: #FFF;
-        margin: 15px;
-        padding-top: 5px;
-        overflow: hidden;
-        white-space: nowrap;
       }
 
       .reminder-footer {
-        font-family: Monaco, monospace;
-        font-size: 14px;
-        color: #FFF;
-        padding-top: 50px;
+        font-family: 'Open Sans', Arial, sans-serif;
+        font-size: 1.0em;
+        font-style: italic;
+      }
+
+      a {
+        color: #484848
+        text-decoration: #484848 wavy underline;
       }
     </style>
   </head>
   <body>
     <div class="reminder">
-      <div class="reminder-names">
-        Dear ${to_name},
+      <div class="reminder-header">
+        <h1>Hello!</h1>
       </div>
       <div class="reminder-text">
-          It is time to move your car!
+        <p>Just a friendly reminder that the scheduled cleaning time for your parked car at <a href="https://www.google.com/maps/search/?api=1&query=${location.lat},${location.lng}">${location.name}</a> is approaching:</p>
```

Comment:
We'll need to figure out a good way of selecting the "middle" of a parking location for the UI, so we can just send this along if the user is unsure of where a specific named location is.